addons/csi/azure-file/driver.yaml

```diff
@@ -32,6 +32,9 @@
 {{ if eq .Cluster.MajorMinorVersion "1.31" }}
 {{ $version = "v1.30.5" }}
 {{ end }}
+{{ if eq .Cluster.MajorMinorVersion "1.32" }}
```

Comment:
Same, please `make` this addon.

Reply:
You can rebuild all addons quick&dirtily using

```shell
cd addons
find . -type d -exec make -C {} \;
```

Reply:
done!